analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.R

```diff
@@ -0,0 +1,81 @@
+# K S Gaonkar 2020
```

Comment:
I had to renumber this module and, because the crux of the issue we are addressing here required logic changes, alter the code. This looks like a brand new script; it is not. The lines that need the closest review are lines 37-48, where the logic for including/excluding samples for subtyping is located.